pkg/server/server.go

```diff
@@ -46,6 +46,7 @@ func configureApp(ctx context.Context, cfg *config.Config) (*fiber.App, error) {
 	}))
 	app.Use(func(c *fiber.Ctx) error {
 		c.SetUserContext(ctx)
+
```

Comment:
@jgiannuzzi this probably happens because of `golangci-lint-full`, should we not having this again for changed files only?